ecl/resource_ecl_compute_instance_v2.go

```diff
@@ -457,6 +466,8 @@ func resourceComputeInstanceV2Read(d *schema.ResourceData, meta interface{}) err
 		return fmt.Errorf("Invalid power_state for instance %s: %s", d.Id(), server.Status)
 	}
 
+	d.Set("config_drive", server.ConfigDrive)
```

Comment:
Terraform schema is bool type, but server.ConfigDrive is a String type, so an error occurs in d.set ().
Could you make the following changes?
```suggestion
	d.Set("config_drive", server.ConfigDrive)
	if server.ConfigDrive == "True" {
		d.Set("config_drive", true)
	} else {
		d.Set("config_drive", false)
	}
```
The reason for doing this is that server.ConfigDrive is "True" when config_drive is true, and "" otherwise.

Reply:
Modification done.